go/genkit/servers.go

```diff
@@ -39,48 +43,39 @@ import (
 	"go.opentelemetry.io/otel/trace"
 )
 
+// StartServer starts the server appropriate for the current environment,
+// listening on addr.
+// StartServer always returns a non-nil error, the one returned by http.ListenAndServe.
+func StartServer(addr string) error {
+	switch e := currentEnvironment(); e {
+	case EnvironmentDev:
+		return StartDevServer(addr)
+	case EnvironmentProd:
+		return StartProdServer(addr)
+	default:
+		return fmt.Errorf("unknown environment value: %q", e)
+	}
```

Comment:
the idea is that `StartProdServer` should always start (regardless of `currentEnvironment()`), even in dev mode. Dev (reflection API) server should only run in dev mode.

So the logic here should be:

```
if currentEnvironment() == EnvironmentDev {
  StartDevServer(addr)
}
StartProdServer(addr)
```

Reply:
Ah, I see. I was going about this wrong.
I removed StartServer and changed StartProdServer to StartFlowServer.
When you call StartFlowServer, you'll get a dev server too.